src/modules/Writers.jl

```diff
@@ -142,7 +144,15 @@ function render(io::IO, mime::MIME"text/plain", node::Expanders.DocsNode, page,
                 """)
             end
         end
-        print(io, "</ul>")
+        print(io, "</ul>\n\n")
+
+        # we print a small notice if we are not displaying all the methods
+        nhidden = length(methodnodes)-length(ms)
+        if nhidden == 1
+            print(io, "_1 method is not being displayed._\n\n")
+        elseif nhidden > 1
+            print(io, "_$(nhidden) methods are not being displayed._\n\n")
+        end
```

Comment:
Can be

```julia
println(io, "$n method$(n == 1 ? "" : "s") not being displayed.\n")
```

to avoid the duplicate text. That's what I've done elsewhere in the package, though up to you if you prefer to keep the current style.